8Knot/assets/body-override.css

```diff
@@ -0,0 +1,33 @@
+/**
+ * Body Background Override
+ * ========================
+ *
```

Comment:
If you are going to transition styling to be in a bunch of different files, put styling in a folder and images in another

Reply:
Actually looking at all of the additions to the assets folder please organize a bit and have more descriptive file naming